src/core/jbig2.js

```diff
@@ -1074,6 +1242,40 @@ var Jbig2Image = (function Jbig2ImageClosure() {
       function SimpleSegmentVisitor_onImmediateLosslessTextRegion() {
       this.onImmediateTextRegion.apply(this, arguments);
     },
+    onPatternDictionary:
+      function SimpleSegmentVisitor_onPatternDictionary(dictionary,
```

Comment:
Since we now support ES6, please just use the following formatting when adding *new* code (applies to the methods below as well):
```javascript
onPatternDictionary(dictionary, currentSegment, data, start, end) {
```